play-frontend-hmrc-play-30/src/main/scala/uk/gov/hmrc/govukfrontend/views/viewmodels/FormGroup.scala

```diff
@@ -19,11 +19,13 @@ package uk.gov.hmrc.govukfrontend.views.viewmodels
 import play.api.libs.functional.syntax._
 import play.api.libs.json._
 import uk.gov.hmrc.govukfrontend.views.viewmodels.content.Content
+import uk.gov.hmrc.govukfrontend.views.viewmodels.content.Content._
 
 case class FormGroup(
   classes: Option[String] = None,
   attributes: Map[String, String] = Map.empty,
-  afterInput: Option[Content] = None
+  afterInput: Option[Content] = None,
+  beforeInput: Option[Content] = None
```

Comment:
can these be swapped around?  I know `afterInput` was added first, but it's just a bit weird them not being in their natural order.  I doubt anyone is already using `afterInput` so wouldn't expect it to break anything

Reply:
😅 🎲 probably is ok